lib/generamba/cli/setup_command.rb

```diff
@@ -33,21 +33,29 @@ def setup
       targets_prompt = ''
       project.targets.each_with_index { |element, i| targets_prompt += ("#{i}. #{element.name}" + "\n") }
       project_target = ask_index("Select the appropriate target for adding your MODULES (type the index):\n" + targets_prompt,project.targets)
-      test_target = ask_index("Select the appropriate target for adding your TESTS (type the index):\n" + targets_prompt,project.targets)
+      include_tests = yes?('Do you want to generate tests? (yes/no)')
```

Comment:
I think that it would be more explicit to ask something like "Are you using unit-tests in this project?"